netdata-installer.sh

```diff
@@ -747,9 +747,9 @@ detect_libc() {
     echo >&2 " Detected musl"
     libc="musl"
   else
-    ls_path=$(command -v ls)
-    if [ -n "${ls_path}" ] ; then
-      cmd=$(ldd "$ls_path" | grep -w libc | cut -d" " -f 3)
+    ln_path=$(command -v ln)
+    if [ -n "${ln_path}" ] ; then
+      cmd=$(ldd "$ln_path" | grep -w libc | cut -d" " -f 3)
```

Comment:
I'd expect us to be able to do just a `cmd=$(ldd /bin/sh | grep -w libc | cut -d" " -f 3)`, because the script itself runs under `/bin/sh`.

Reply:
I would tend to agree here. We really need to just be looking at a ‘well known’ absolute path that we can functionally guarantee points to an actual executable, because that’s essentially the only way we can reliably avoid aliases interfering here. `/bin/sh` is the most sensible option for that given that the script itself runs under `/bin/sh`.

In fact, if we do it that way, then this check will work correctly on a much wider variety of systems than the earlier `ldd` checks, and I would argue that performance is sufficiently non-critical here to warrant just putting this check first (possibly after the check for musl) as well...

Reply:
Ok, changed to use /bin/sh. It should fix the linked issue, I suppose we can revisit in general if needed.